src/python/ensembl/io/genomio/integrity.py

```diff
@@ -733,10 +732,7 @@ def main() -> None:
     """Main entrypoint."""
     mod = argschema.ArgSchemaParser(schema_type=InputSchema)
 
-    # Start
-    inspector = IntegrityTool(mod.args["manifest_file"])
-    if mod.args.get("brc_mode"):
-        inspector.set_brc_mode(True)
+    inspector = IntegrityTool(mod.args["manifest_file"], mod.args["brc_mode"])
```

Comment:
Better be explicit in case we want to make changes later:
```suggestion
    inspector = IntegrityTool(mod.args["manifest_file"], brc_mode=mod.args["brc_mode"])
```